spec/support/hanami_commands.rb

```diff
@@ -15,6 +15,7 @@ def rackup(args = {}, &blk)
       end
 
       def server(args = {}, &blk)
+        args[:port] ||= rand(2300..9999)
```

Comment:
I don't see the advantage here... Prime is costly and the select is costly...

As Prime will only return the prime numbers, there will be fewer ports to choose from, so a greater risk of it being used, isn't it?

Couldn't we instead just check if the port is being used and take another random port if it is?